### PR TITLE
feat(docs): Fix sections not rendering on Read the Docs

### PR DESCRIPTION
**Pull request by yavorona**
Summary
-------

 - Added jsonschema dependency to fix rendering issues.
 - Included readme.md directly to index.rst instead of having readme.rst.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,11 +1,4 @@
-Optimizely Python SDK
-=====================
-
-.. toctree::
-   :caption: Introduction
-
-   readme
-
+.. mdinclude:: ../../README.md
```
Comment (aliabbasrizvi) [inline docs/source/index.rst:1]: Yep, this was the change I needed.

File: requirements/docs.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 sphinx==2.4.4
 sphinx-rtd-theme==0.4.3
-m2r==0.2.1
\ No newline at end of file
+m2r==0.2.1
+jsonschema==3.2.0
```
Comment (aliabbasrizvi) [inline requirements/docs.txt:4]: Why is jsonschema needed?

Comment (aliabbasrizvi) [inline requirements/docs.txt:4]: I think it should be removed.

Comment (yavorona) [reply requirements/docs.txt:4]: Talked offline. jsonschema is needed to render docstrings from `optimizely`, `config_manager` , and `optimizely_config`.

Comment (yavorona) [reply requirements/docs.txt:4]: Talked offline. jsonschema is needed to render docstrings from `optimizely`, `config_manager` , and `optimizely_config`.

**Comment by aliabbasrizvi** [comment]
Looks like tests passed: https://travis-ci.org/github/optimizely/python-sdk/builds/687595137

Going to force merge this.

**Pull request merged**